frontend/src/views/ProjectNotFound.vue

```diff
@@ -14,46 +14,30 @@ See the License for the specific language governing permissions and
 limitations under the License.
  -->
 <template>
-  <v-container fluid class="fill-height text-center">
-    <v-row align="center">
-      <v-col>
-        <h1>404</h1>
-        <h2>Project not found</h2>
-        <p class="message">The project you are looking for doesn't exist or an other error occured!</p>
-        <v-btn dark color="cyan darken-2" @click="goBack">
-          Get me out of here
-        </v-btn>
-      </v-col>
-    </v-row>
-  </v-container>
+  <g-error
+    code="404"
+    text="Project not found"
+    message="The project you are looking for doesn't exist or an other error occured!"
```

Comment:
another or an other?

Reply:
It is not another error. It is an error other than not found.